### PR TITLE
Add contributing.md file

### LINKED ISSUE #2: Review Process
For reviewing we follow this process.

- Add the list of all the section names (in english) in the pull request description, which reviewer will mark check as and when the review of each section is getting completed.
- You should not resolve review feedback, you can add a comment or an any (👍) emoji to every review comment once it's resolved and pushed.
- A reviewer will review the changes and resolve the conversations

> Note: Minimum one reviewers approval needed to get the PR merged

### PR DESCRIPTION
**Pull request by arshadkazmi42**
Fixes #2 
-

### CODE REVIEW

File: contibuting.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+git push origin नयी-ब्रांच-का-नाम
+```
+
+# रिव्यु प्रोसेस
+
+हर पुल्ल रिक्वेस्ट को २ चरण के रिव्यु को पार करना ज़रूरी है। रिव्यु के लिए हम निचे दी गयी बातों का पबातों का पालन करें।
+
+- सरे वर्गों का नाम अंग्रेज़ी में पुल्ल रिक्वेस्ट के विवरण में लिखें। जिसको रीवीउअर रिव्यु पूरा होने के बाद चेक मार्क करेगा।
+- किसी भी रीवीउअर के रीव्यु फीडबैक को रीसोलव मार्क न करें। रीवीउअर आपके फीडबैक फिक्सेस को देखने के बाद उसको रीसोलव मार्क करेगा।
+- आप फीडबैक के चंगेस करने के बाद उस फीडबैक पर कमेंट डालें या ये 👍 (इमोजी) लगा दें जिससे रीवीउअर को पता चल सके की आपने इस फीडबैक का फिक्स कर दिया है।
```
Comment (SatyaRF) [inline contibuting.md]: के चंगेस => में परिवर्तन

**Review by SatyaRF — CHANGES_REQUESTED** [review]

File: contibuting.md (hunk 1 above)
Comment (arshadkazmi42) [reply contibuting.md]: @SatyaRF Thanks you for reviewing. Fixed this.

**Comment by arshadkazmi42** [comment]
@SatyaRF Can you review this and Approve it if everything is fine?
Also there are two more open PRs if you could review those also

**Review by SatyaRF — APPROVED** [review]

**Comment by arshadkazmi42** [comment]
@SatyaRF Thank you for the review. Can you also review the other two open PRs?
Also, i have reviewed your PR and added some comments, can you make the fixes for those also.

**Pull request merged**